README.md

```diff
@@ -1,3 +1,24 @@
-# NewXKitBot
+# New-XKit Bot
+[![Gitter](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/new-xkit/NewXKitBot?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
 
-[![Gitter](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/new-xkit/NewXKitBot?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
\ No newline at end of file
+### What is New-XKit Bot?
+New-XKit Bot is the team behind [New XKit](https://github.com/new-xkit/XKit)'s helping little assistant, written in JavaScript for [Node.js](https://nodejs.org/). It's here to make the lives of the New-XKit team just a tiny bit easier!
```

Comment:
This could be misread as "(New-XKit Bot is the team behind New XKit)'s helping little assistant" rather than "New-XKit Bot is (the team behind New XKit)'s helping little assistant". For clarity, I would rewrite this as

> New-XKit Bot is New XKit team's little assistant